lib/process-services/task-list/components/task-list.component.ts

```diff
@@ -82,23 +83,13 @@ export class TaskListComponent implements OnChanges, AfterContentInit, Paginated
     @Input()
     landingTaskId: string;
 
-    /** Data source object that represents the number and the type of the columns that
-     * you want to show.
-     */
-    @Input()
-    data: DataTableAdapter;
-
     /** Row selection mode. Can be none, `single` or `multiple`. For `multiple` mode,
      * you can use Cmd (macOS) or Ctrl (Win) modifier key to toggle selection for
      * multiple rows.
      */
     @Input()
     selectionMode: string = 'single'; // none|single|multiple
 
-    /** Custom preset column schema in JSON format. */
-    @Input()
-    presetColumn: string;
```

Comment:
we can not remove those input you can deprecate it

Reply:
OK

Reply:
I don't get what we are doing. We are removing this input but we have a static property in task list 
```    
static PRESET_KEY = 'adf-task-list.presets';
```
and I don't get what datatable schema is using

```
 @Input()
 presetColumn: string;

customSchemaColumns = this.getSchemaFromConfig(this.presetColumn)
```

Reply:
PRESET_KEY:  it holds the key to fetch the presets.
https://github.com/sivakumar414ram/alfresco-ng2-components/blob/04bfd505cef92dc61bbf95d960015d8172975dc5/lib/core/datatable/data/data-table.schema.ts#L50

@Input() presetColumn : it holds the key to be picked from available presets.
https://github.com/sivakumar414ram/alfresco-ng2-components/blob/04bfd505cef92dc61bbf95d960015d8172975dc5/lib/core/datatable/data/data-table.schema.ts#L75